parsons/scytl/scytl.py

```diff
@@ -0,0 +1,664 @@
+import urllib
+import zipfile
+import csv
+import requests
+import urllib.error
+import urllib.request
+import xml.etree.ElementTree as ET
+from datetime import datetime
+from dateutil.parser import parse as parsedate
+from pytz import timezone
+from io import BytesIO, StringIO
+
+clarity_url = 'https://results.enr.clarityelections.com/'
+
+get_version_url = \
+    clarity_url + '{administrator}/{election_id}/current_ver.txt'
+summary_csv_url_template = \
+    clarity_url + '{administrator}//{election_id}/{version_num}/reports/summary.zip'
+detail_xml_url_template = \
+    clarity_url + '{administrator}/{election_id}/{version_num}/reports/detailxml.zip'
+county_detail_xml_url = clarity_url + \
+    '{state}/{county_name}/{county_election_id}/{county_version_num}/reports/detailxml.zip'
+detail_settings_json_url_template = \
+    clarity_url + '{state}/{election_id}/{version_num}/json/en/electionsettings.json'
+
+tzinfos = {
+    "EST": "UTC-5",
+    "EDT": "UTC-4",
+    "CST": "UTC-6",
+    "CDT": "UTC-5",
+    "MST": "UTC-7",
+    "MDT": "UTC-6",
+    "PST": "UTC-8",
+    "PDT": "UTC-7",
+    "AKST": "UTC-9",
+    "AKDT": "UTC-8",
+    "HST": "UTC-10",
+    "HDT": "UTC-9",
+}
+
+
+class CountyDetails:
```

Comment:
This could be a dataclass.

Reply:
I'm of two minds about this. Turning it into a dataclass will improve readability for some, but may make the code harder to read and reason about for many who are newer to Python and unfamiliar with dataclasses. Perhaps we can switch to using a dataclass if you're up for it @agreenspan24 and put a code comment for readers linking to an explanation of dataclasses ie https://docs.python.org/3/library/dataclasses.html

Reply:
Added as a dataclass with comment